CIFAR10/train/diux_cifar_g3.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "v1.0",
+    "author": "DIUX",
+    "authorEmail": "ybulatov.ctr@diux.mil",
+    "framework": "tensorflow 1.5, tensorpack 0.8.1",
+    "codeURL": "https://github.com/ppwwyyxx/tensorpack/blob/master/examples/ResNet/cifar10-resnet.py",
+    "model": "ResNet50",
+    "hardware": "g3.4xlarge",
+    "timestamp": "2018-01-31",
+    "costPerHour": 1.14,
+    "optimizer": "SGD with Momentum",
+    "momentum": 0.9,
+    "schedule": [
+        {"learning_rate": 0.1, "epochs": 1},
+        {"learning_rate": 0.01, "epochs": 82},
+        {"learning_rate": 0.001, "epochs": 123},
+        {"learning_rate": 0.0002, "epochs": 300}
```

Comment:
The format of the CIFAR10 and ImageNet schedules are inconsistent. Here epochs seem to mean the epoch where the change in learning rate occurs. More concretely, are you saying the learning schedule is the following?

1. Learning rate = 0.1 from epoch 1 to epoch 82
2. Learning rate = 0.01 from epoch 83 to 123
3. Learning rate = 0.001 from epoch 124 to 300
4. Learning rate = 0.0002 from epoch 300 to 400, which is the end of the TSV

The ImageNet JSON uses epochs as the number of epochs for the specified learning rate, so adding up the epochs fields in schedule adds up to the total number of epochs in the TSV. I prefer the approach from the ImageNet JSON. I think this should be:

```
        "schedule": [
                {"learning_rate": 0.1, "epochs": 82},
                {"learning_rate": 0.01, "epochs": 41},
                {"learning_rate": 0.001, "epochs":  177},
                {"learning_rate": 0.0002, "epochs": 100}
            ]
```

82 + 41 + 177 + 100 == 400 matching the TSV